app/views/manageBusinesses/add/AddIncomeSources.scala.html

```diff
@@ -48,8 +48,8 @@
 }
 
 @addBusinessNameURL = @{
-    if(isAgent) controllers.manageBusinesses.add.routes.AddBusinessNameController.showAgent().url
-    else controllers.manageBusinesses.add.routes.AddBusinessNameController.show().url
+    if(isAgent) controllers.manageBusinesses.add.routes.AddBusinessNameController.show(isAgent = true, isChange = false).url
```

Comment:
does this work?
'
@addBusinessNameURL = @{ controllers.manageBusinesses.add.routes.AddBusinessNameController.show(isAgent = isAgent, isChange = false).url
'

Reply:
No cause isChange is not passed here so it should be set manually to false

Reply:
it is being set manually to false: `isChange = false`

Reply:
Again my bad, it works in that way yes👌🏻